<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainGUI</class>
 <widget class="QMainWindow" name="MainGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1028</width>
    <height>769</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CTC</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QTabWidget" name="line_tabs">
        <property name="styleSheet">
         <string notr="true">QLabel:hover{border: 1px solid wheat}</string>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="blue_line_tab">
         <attribute name="title">
          <string>Blue Line</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="10">
           <widget class="QLabel" name="block7">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>5</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>5</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(153, 193, 241);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLabel" name="block3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>5</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>5</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(153, 193, 241);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="9">
           <widget class="QLabel" name="block11">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>5</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>5</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(153, 193, 241);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="13">
           <widget class="QLabel" name="block15">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>5</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>5</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(51, 209, 122);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="9">
           <widget class="QLabel" name="block6">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>5</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>5</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(153, 193, 241);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="13">
           <widget class="QLabel" name="block10">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>5</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>5</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(51, 209, 122);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="block2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>5</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>5</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(153, 193, 241);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="block1">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>5</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>5</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(153, 193, 241);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="11">
           <widget class="QLabel" name="block13">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>5</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>5</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(153, 193, 241);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="8" rowspan="2">
           <widget class="QLabel" name="track_switch">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>5</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>5</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="sizeIncrement">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>5</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(0, 0, 0);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="11">
           <widget class="QLabel" name="block8">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>5</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>5</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(153, 193, 241);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="10">
           <widget class="QLabel" name="block12">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>5</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>5</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(153, 193, 241);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="12">
           <widget class="QLabel" name="block9">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>5</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>5</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(153, 193, 241);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="5">
           <widget class="QLabel" name="block5">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>5</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>5</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(153, 193, 241);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="0" rowspan="3">
           <widget class="QLabel" name="Yard">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(0, 0, 0);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QLabel" name="block4">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>5</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>5</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(153, 193, 241);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="12">
           <widget class="QLabel" name="block14">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>5</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>5</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(153, 193, 241);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="red_line_tab">
         <attribute name="title">
          <string>Red Line</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="green_line_tab">
         <attribute name="title">
          <string>Green Line</string>
         </attribute>
        </widget>
       </widget>
       <widget class="QTableWidget" name="tableWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <row>
         <property name="text">
          <string/>
         </property>
        </row>
        <row>
         <property name="text">
          <string/>
         </property>
        </row>
        <column>
         <property name="text">
          <string>Property</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Value</string>
         </property>
        </column>
        <item row="0" column="0">
         <property name="text">
          <string>Block Name</string>
         </property>
        </item>
        <item row="0" column="1">
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item row="1" column="0">
         <property name="text">
          <string>Status</string>
         </property>
        </item>
        <item row="1" column="1">
         <property name="text">
          <string>Unocupied</string>
         </property>
        </item>
       </widget>
      </widget>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QLineEdit" name="lineEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>28</height>
         </size>
        </property>
       </widget>
       <widget class="QTextBrowser" name="textBrowser"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1028</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSchedule">
    <property name="title">
     <string>Schedule</string>
    </property>
    <addaction name="actionImport_Schedule"/>
    <addaction name="actionExport_Schedule"/>
    <addaction name="actionEdit_Schedule"/>
   </widget>
   <widget class="QMenu" name="menuMode">
    <property name="title">
     <string>Mode</string>
    </property>
    <addaction name="actionAutomatic_2"/>
   </widget>
   <widget class="QMenu" name="menuDebug">
    <property name="title">
     <string>Debug</string>
    </property>
    <addaction name="actionInput_Debug"/>
    <addaction name="actionOutput_Debug"/>
   </widget>
   <addaction name="menuSchedule"/>
   <addaction name="menuMode"/>
   <addaction name="menuDebug"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionImport_Schedule">
   <property name="text">
    <string>Import Schedule</string>
   </property>
  </action>
  <action name="actionExport_Schedule">
   <property name="text">
    <string>Export Schedule</string>
   </property>
  </action>
  <action name="actionEdit_Schedule">
   <property name="text">
    <string>Edit Schedule</string>
   </property>
  </action>
  <action name="actionFixed">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Fixed</string>
   </property>
  </action>
  <action name="actionAutomatic">
   <property name="text">
    <string>Automatic</string>
   </property>
  </action>
  <action name="actionAutomatic_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Automatic</string>
   </property>
  </action>
  <action name="actionInput_Debug">
   <property name="text">
    <string>Input Debug</string>
   </property>
  </action>
  <action name="actionOutput_Debug">
   <property name="text">
    <string>Output Debug</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
